packages/composite-checkout/src/wpcom/wp-order-review-line-items.js

```diff
@@ -130,7 +122,7 @@ function DeleteIcon( { uniqueID } ) {
 			xmlns="http://www.w3.org/2000/svg"
 			aria-labelledby={ uniqueID }
 		>
-			<title id={ uniqueID }>{ localize( 'Delete' ) }</title>
+			<title id={ uniqueID }>{ localize( 'Remove ' + product + ' from cart' ) }</title>
```

Comment:
I'm not 100% sure if this will work right this way.

Reply:
It won't. You'll want it to be: `sprintf( localize( 'Remove %s from cart' ), product )` and you might have to import `sprintf` from `lib/localize`.